env0/resource_module.go

```diff
@@ -51,7 +51,7 @@ func resourceModule() *schema.Resource {
 			},
 			"token_id": {
 				Type:         schema.TypeString,
-				Description:  "the token id used for integration with GitLab",
+				Description:  "the token id used for integration with GitLab, Azure Devops, and more...",
```

Comment:
This isn't 100% correct because it's also used as a simple git token. I would simple say:
```suggestion
				Description:  "the git token id to be used",